scapy/arch/pcapdnet.py

```diff
@@ -40,7 +40,9 @@ def winpcapy_get_if_list():
 
   except OSError as e:
     if conf.interactive:
-      log_loading.error("Unable to import libpcap library: %s" % e)
+      log_loading.warning("wpcap.dll is not installed. You won't be able to send/recieve packets. Visit the scapy's doc to install it")
+      def winpcapy_get_if_list():
```

Comment:
Do you define `winpcapy_get_if_list` only in interactive mode for a particular reason? I feel like it would be useful also when imported.